sklearn/feature_extraction/text.py

```diff
@@ -1585,10 +1584,9 @@ class TfidfVectorizer(CountVectorizer):
         out of the raw, unprocessed input.
 
         .. versionchanged:: 0.21
-
-        Since v0.21, if ``input`` is ``filename`` or ``file``, the data is
-        first read from the file and then passed to the given callable
-        analyzer.
+            Since v0.21, if ``input`` is ``filename`` or ``file``, the data is
+            first read from the file and then passed to the given callable
+            analyzer.
```

Comment:
```suggestion
            Since v0.21, if ``input`` is ``'filename'`` or ``'file'``, the data
            is first read from the file and then passed to the given callable
            analyzer.
```